src/platform/user/profile/vet360/components/PhoneField/VAPPhoneView.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import PhoneNumberWidget from 'platform/forms-system/src/js/review/PhoneNumberWidget';
+
+export default function VAPPhoneView({ data: phoneData }) {
+  const phoneNumber = (
+    <PhoneNumberWidget
+      value={[phoneData.areaCode, phoneData.phoneNumber].join('')}
+    />
+  );
+
+  const extension = phoneData.extension && <span>x{phoneData.extension}</span>;
+  return (
+    <div>
+      {phoneNumber} {extension}
```

Comment:
On Profile 2.0, we want to remove the international prefix since we only accept US phone numbers. I created a new component rather than passing a prop to the existing `PhoneView` component that would determine whether we are viewing Profile 2.0, so it will be easy to remove the `PhoneView` when we make the full transition to Profile 2.0.

Reply:
For consistency's purposes, we keep the (xxx) xxx-xxx formatting for phone numbers, but we do want to remove the international prefix. Confirmed with Tressa.

Reply:
I'm kind of surprised that `extension` isn't an optional prop for `<PhoneNumberWidget>` - I wonder if that's something worth adding eventually.